test/tools/ossfuzz/StackReuseCodegenFuzzer.cpp

```diff
@@ -47,6 +50,24 @@ using namespace std;
 
 static evmc::VM evmone = evmc::VM{evmc_create_evmone()};
 
+namespace
+{
+/// @returns true if there are no recursive functions, false otherwise.
```

Comment:
```suggestion
/// @returns true if there are recursive functions, false otherwise.
```
Or the logic of the function should be inverted, since you are returning `true` when you find a recursive function.